static/app/bootstrap/exportGlobals.tsx

```diff
@@ -47,6 +47,8 @@ const SentryApp = {
   getModalPortal: require('sentry/utils/getModalPortal').default,
   Client: require('sentry/api').Client,
   IconArrow: require('sentry/icons/iconArrow').IconArrow,
+  IconClose: require('sentry/icons/iconClose').IconClose,
+  IconCheckmark: require('sentry/icons/iconCheckmark').IconCheckmark,
```

Comment:
@roggenkemper how hard would it be to export all the icons in one line?

Reply:
Probably best to just export the ones we need to keep the bundle from getting too big